spec/std/process_spec.cr

```diff
@@ -237,4 +246,54 @@ describe Process do
       Process.find_executable("some_very_unlikely_file_to_exist").should be_nil
     end
   end
+
+  # This doesnt work in the linux test environment due to the test environment
+  # FIXME: Discover the reason why `LibC.getgrgid(2000)` does not return a user.
+  {% if !flag?(:linux) %}
```

Comment:
`if !exp` => `unless exp`

Reply:
Yes they are functionally equivalent. Base uses !flag? and I personally prefer traditional `if not` to `unless`.  Not changing.

Reply:
We try to keep logical branches always positive, hence such comments.

Reply:
I'm sorry to be harsh, but this isn't your project, this is an open source project. Standards exist for a reason.

Reply:
I would happily stick to the standard if there was one.
```
grep "{% if \!" **/*.cr
spec/compiler/codegen/thread_local_spec.cr:{% if !flag?(:openbsd) %}
spec/std/http/client/client_spec.cr:      {% if !flag?(:without_openssl) %}
spec/std/process_spec.cr:  {% if !flag?(:linux) %}
src/atomic.cr:    {% if !T.union? && (T == Char || T < Int::Primitive || T < Enum) %}
src/benchmark.cr:    {% if !flag?(:release) %}
src/benchmark.cr:    {% if !flag?(:release) %}
src/file.cr:      {% if !flag?(:windows) %}
src/http/client.cr:    {% if !flag?(:without_openssl) %}
src/http/client.cr:{% if !flag?(:without_openssl) %}
src/http/common.cr:{% if !flag?(:without_zlib) %}
src/http/server.cr:{% if !flag?(:without_openssl) %}
src/http/server.cr:  {% if !flag?(:without_openssl) %}
src/http/server.cr:    {% if !flag?(:without_openssl) %}
src/http/server/handlers/compress_handler.cr:{% if !flag?(:without_zlib) %}
src/http/web_socket/protocol.cr:{% if !flag?(:without_openssl) %}
src/http/web_socket/protocol.cr:    {% if !flag?(:without_openssl) %}
```

Reply:
Fair enough, this is fine then we can have a future PR to bring us all up to spec.